Dogecoin Turkish Languges translation for  Website

emrullahsaku opened the pull request:
Dogecoin Turkish Languges translation for  Website is completed. Please review and accept my request.

### Description
<!-- Describe your changes in detail -->
I can translate in Turkish

### Motivation and Context
<!-- Why is this change required? What problem does it solve? -->
<!-- If it fixes an open issue, please link to the issue here. -->
I like the Dogecoin. I really would like to contribute the Dogecoin community as a volunteer.


### How Has This Been Tested?
<!-- Please describe in detail how you tested your changes. -->
<!-- Include details of your testing environment, tests ran to see how -->
<!-- your change affects other areas of the code, etc. -->
just Turkish Language translate.


### Screenshots (if appropriate):

### Checklist:
<!-- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] GitHub issue linked
- [ ] The necessary translations have been added for all languages
- [ ] Any documentation has been updated accordingly
- [ ] Responsive sizes (Mobile) tested
- [ ] Cross Browser tested if necessary
- [ ] Conflicts resolved

cihanbarisgoren commented on guide.html:
```diff
@@ -9,7 +9,7 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
-    <meta name="description" content="Dogecoin is an open source peer-to-peer digital currency, favored by Shiba Inus worldwide." />
+    <meta name="description" content="Dogecoin, dünya çapında Shiba Inus tarafından tercih edilen, açık kaynaklı bir eşler arası dijital para birimidir." />
```
*Shiba Inular tarafından tercih edilen

cihanbarisgoren commented on guide.html:
```diff
@@ -27,9 +27,9 @@
     <meta property="og:url" content="https://dogecoin.com/" />
     <meta property="og:title" content="Dogecoin| Getting started Guide" />
-    <meta property="og:description" content="Dogecoin is an open source peer-to-peer digital currency, favored by Shiba Inus worldwide." />
+    <meta property="og:description" content="Dogecoin, dünya çapında Shiba Inus tarafından tercih edilen, açık kaynaklı bir eşler arası dijital para birimidir." />
```
*Shiba Inular tarafından tercih edilen

cihanbarisgoren commented on index.html:
```diff
@@ -3,7 +3,7 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
-    <meta name="description" content="Dogecoin is an open source peer-to-peer digital currency, favored by Shiba Inus worldwide." />
+    <meta name="description" content="Dogecoin, dünya çapında Shiba Inus tarafından tercih edilen, açık kaynaklı bir eşler arası dijital para birimidir." />
```
*Shiba Inular tarafından tercih edilen

langerhans commented:
Hi, and thank you for the contribution! To be able to merge this you'd need to do the translation work in the i18n files. Check this directory: https://github.com/dogecoin/dogecoin.com/tree/gh-pages/assets/i18n . Copy e.g. en.json as tr.json and make your changes there. See for example #184 on how that works exaclty.

emrullahsaku commented:
Hi, I am glad to to contribute to Dogecoin community to publicize.
I have created tr.json language and translated.

I have committed on github.

Thanks

On Mon, May 10, 2021 at 8:31 PM Max K. ***@***.***> wrote:

> Hi, and thank you for the contribution! To be able to merge this you'd
> need to do the translation work in the i18n files. Check this directory:
> https://github.com/dogecoin/dogecoin.com/tree/gh-pages/assets/i18n . Copy
> e.g. en.json as tr.json and make your changes there. See for example #184
> <https://github.com/dogecoin/dogecoin.com/pull/184> on how that works
> exaclty.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/dogecoin/dogecoin.com/pull/183#issuecomment-837013173>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUAJ4TTW6PPGD5RP6NDNTV3TNAKA5ANCNFSM44PXLUXQ>
> .
>

langerhans commented:
Thank you! Now you'd still need to revert the changes to the html files where you replaced the English text with Turkish. Then we can merge it :)

emrullahsaku commented:
Okay, I reverted the changes to the html files where I replaced the English text with Turkish.
Thank you :)

> Thank you! Now you'd still need to revert the changes to the html files where you replaced the English text with Turkish. Then we can merge it :)

langerhans commented:
Thanks, I'll merge your changes and then add some fixes for it to shop up correctly. Should be visible later today :)

emrullahsaku commented:
> Thanks, I'll merge your changes and then add some fixes for it to shop up correctly. Should be visible later today :)

Thank you. I saw it is live now. Great :)

Aivolt1 reviewed (commented):
Complete

emrullahsaku commented:
Hi, I wonder that can my name added for contributors page. Thanks
